api/src/api/agencies_v1/agency_schema.py

```diff
@@ -102,6 +105,12 @@ class AgencyV1Schema(Schema):
             "example": "False",
         },
     )
+    is_test_agency = fields.Boolean(
+        metadata={
+            "description": "Indicates if the agency is a test agency .",
+            "example": "False",
```

Comment:
While it works because of the JSON parsing library we have, this (and the example added for `has_active_opportunity` above) mean the examples we end up putting in openapi has a value of `"False"` which is only valid in python and isn't actually valid JSON. just pass in a boolean like `False` instead of `"False"`
```suggestion
            "example": False,
```